client/dataset_test.go

```diff
@@ -12,12 +12,23 @@ func TestDatasets(t *testing.T) {
 
 	c := newTestClient(t)
 	datasetName := testDataset(t)
+	updatedDescription := "buzzing with data"
+	updatedExpandJSONDepth := 3
 
 	currentDataset := &Dataset{
-		Name: datasetName,
-		Slug: urlEncodeDataset(datasetName),
+		Name:            datasetName,
+		Description:     &updatedDescription,
```

Comment:
I think if you remove `Description` and `ExpandJSONDepth` from this `currentDataset` initialization you will get passing tests.

Because we can't delete datasets via the API today, you should be sure to have a cleanup function that resets the dataset to the default values as well

Reply:
thanks for the comments, will be getting back to addressing these changes today